### DISCUSSION TITLE
Future of this repository

### DISCUSSION THREAD

**Discussion by Schievel1** [Antynea/grub-btrfs#217] [General]
Hello folks,

I really hope @Antynea is doing ok and well. The last time he did something on GitHub was half a year ago and the last time he committed something to this repository was 9 months ago. 
It is totally understandable to abandon a project if the privat life doesn’t allow to work on it. I just hope he’s doing well and nothing bad happened to him. 

That said I wanted to discuss future of this repository with you guys if Antynea is not coming back any time soon. (And if he comes back but doesn’t want to work on this anymore)
I think we got 3 options here:
- Antynea gives someone write access to this repo so that someone can merge PR and continue development. 
- Someone forks the repo and continues working on it. I would do that, but I don’t think I am such a shell script magician yet that I could do it. I also have no clue about systemd. 
- we create a organisation, fork the repo and work on it as an organisation. I would be happy to help with openrc here and maybe also do a fix or two on the shell script.

I would be happiest with the first thing, but I don’t know if Antynea comes back at all. (To give one of us write access)
Let me know what you think.

**Comment by Antynea** [comment]
Hello,

I want you to know that I am fine.
I no longer have the motivation to continue this project.

I am here to accept your PR, but indeed having a collaborator with access to writing on this project and especially motivated would be welcome.

**Reply by Schievel1** [reply]
Glad to hear you are good. I was thinking maybe we could split the work into several parts. I am working in the daemon right now, but I think I am just not fluent enough in shell scripts to work on the main thing. Maybe one more person is interested in that?

**Reply by imthenachoman** [reply]
So is this repo still under active development now?

**Reply by Schievel1** [reply]
> So is this repo still under active development now?

Yes it is. It's just that I don't have many things to do with it right now
